en/deploy-on-gcp-gke.md

```diff
@@ -66,11 +66,11 @@ gcloud config set compute/region <gcp-region>
 
     ```shell
     gcloud container node-pools create pd --cluster tidb --machine-type n2-standard-4 --num-nodes=1 \
-        --node-labels=dedicated=pd --node-taints=dedicated=pd:NoSchedule
+        --node-labels=dedicated=pd --node-taints=dedicated=pd:NoSchedule --region us-east1
```

Comment:
we have a `Configure your GCP project and default region` step above, but we also added `--region` in `gcloud container clusters create tidb --region us-east1 --machine-type n1-standard-4 --num-nodes=1`. Can you add a comment to explain it?